### ISSUE TITLE
occ: command not found

### ISSUE THREAD

**Issue by sunjam** [nextcloud/nextcloudpi#321]
`ls` shows that occ is in the assumed directory, but here is output when I attempt to run nc-info from the terminal.  [Also, someone else is posting about the occ command not found in this help thread.](https://help.nextcloud.com/t/nextcloudpi-error-sudo-var-www-nextcloud-occ-command-not-found/24462/2)
```
Launching nc-info
cat: /usr/local/etc/ncp-baseimage: No such file or directory
sudo: /var/www/nextcloud/occ: command not found

```

**Comment by nachoparker** [comment]
can you verify that `occ` has no executable right? how could this happen?

**Comment by nachoparker** [comment]
I suspect that this could have happened after upgrading to NC 12.0.4... is this possible?

**Comment by nachoparker** [comment]
I am pretty sure that's what happened (look at other thread)

will fix soon

**Comment by nachoparker** [comment]
fixed in `v0.41.9`

**Comment by rahul606** [comment]
Hi, while running bash talk.sh command I having this error:

> sudo: /var/www/nextcloud/occ: command not found
> /dev/fd/63: line 631: [: : integer expression expected
> /dev/fd/63: line 667: [: : integer expression expected
> Could not open input file: /var/www/nextcloud/occ
> Sorry but something went wrong. Please report this issue to https://github.com/nextcloud/vm/issues and include the output of the error message. Thank you!
> sudo -u www-data php /var/www/nextcloud/occ -V failed

Just to mention I have installed the NextCloud in a different location on my server as: /var/www/abc/html/ 

So, command is looking for occ file in a location which does not exist in my server. Please let me know the solution.
